fizz-buzz.test.js

```diff
@@ -1,4 +1,32 @@
-const { fizzBuzz } = require('./fizz-buzz');
+const { fizzBuzz } = require("./fizz-buzz");
+
+describe("give FizzBuzz for multiples of 3 and 5", () => {
```

Comment:
Usually, you'd group several tests together in a single describe() (ie this one, and remove the others in this file). The label for the describe could be the name of the function eg fizzBuzz.
So, one `describe` wrapping the four `it` tests.